tutor/templates/apps/openedx/settings/partials/common_all.py

```diff
@@ -13,7 +13,7 @@
     "password": {% if MONGODB_PASSWORD %}"{{ MONGODB_PASSWORD }}"{% else %}None{% endif %},
     # Connection/Authentication
     "ssl": {{ MONGODB_USE_SSL }},
-    "authSource": "{{ MONGODB_AUTH_SOURCE }}",
+    "authsource": "{{ MONGODB_AUTH_SOURCE }}",
     "replicaSet": {% if MONGODB_REPLICA_SET %}"{{ MONGODB_REPLICA_SET }}"{% else %}None{% endif %},
```

Comment:
```suggestion
    "replicaset": {% if MONGODB_REPLICA_SET %}"{{ MONGODB_REPLICA_SET }}"{% else %}None{% endif %},
```
Looking more if pymongo use insensitive dict https://github.com/mongodb/mongo-python-driver/blob/3.12.3/pymongo/mongo_client.py#L651  we can test changing  also `replicaSet` to `replicaset` in order to have mongo_db_parameters in standard way kwargs using lowercase.

Reply:
It ain't broken (yet) so let's not try to fix this.